.github/workflows/test.yml

```diff
@@ -19,130 +14,132 @@ concurrency:
+            ${{ runner.OS }}-node-
+            ${{ runner.OS }}-
+      - name: Install Packages
+        run: npm install # maybe this should be changed to npm ci
+      - name: Build the TypeScript OpenAPI with tsoa
+        run: npm run build
+      - name: Run tests
+        run: npm run test
 
-      - id: get-packages
-        name: Get all NPM packages
-        run: $(npm bin)/ts-node --cwdMode ./scripts/getPackages.ts
+  check-version:
+    name: "Check version"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: git fetch --depth=1 --tags origin
+      - name: Install yq
+        run: sudo snap install yq
+      - name: Check Build Version
+        id: get_version
+        run: ./scripts/check-version.sh
+        shell: bash
+      - name: Error if version is not increased
+        shell: bash
+        run: |
+          exit $([[ "${{steps.get_version.outputs.IS_NEW_VERSION}}" = "true" ]] && echo 0 || echo 1)
 
-  release-please:
+  build-docker:
+    name: "Build docker image"
     runs-on: ubuntu-latest
-    needs: [package-finder, lint]
-    strategy:
-      fail-fast: false
-      matrix:
-        package: ${{fromJson(needs.package-finder.outputs.packages)}}
     steps:
-      - uses: google-github-actions/release-please-action@v2
-        id: release-please
+      - uses: actions/checkout@v3
+      - name: Setup QEMU
+        uses: docker/setup-qemu-action@v2
         with:
-          path: packages/${{ matrix.package }}
-          token: ${{ secrets.GITHUB_TOKEN }}
-          release-type: node
-          package-name: ${{ matrix.package }}
-          bump-minor-pre-major: true
-          bump-patch-for-minor-pre-major: true
-          monorepo-tags: true
-          signoff: 'github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>'
-
-      # Initiate release process if release was created
-      - name: Checkout veritable-cloudagent
-        uses: actions/checkout@v3
-        if: ${{ steps.release-please.outputs.release_created }}
-      # Some packages need indy-sdk for node as part of npm install
-      - name: Setup Libindy
-        uses: ./.github/actions/setup-libindy
-        if: ${{ steps.release-please.outputs.release_created }}
-      - name: Setup NodeJS
-        uses: ./.github/actions/setup-node
+          platforms: all
+      - name: Setup Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          buildkitd-flags: "--debug"
+      - name: Build image
+        uses: docker/build-push-action@v4
         with:
-          node-version: 18
-        if: ${{ steps.release-please.outputs.release_created }}
-      - name: Install dependencies
-        run: npm ci
-        if: ${{ steps.release-please.outputs.release_created }}
+          builder: ${{ steps.buildx.outputs.name }}
+          context: .
+          file: ./afj-arm.dockerfile
+          # platforms: linux/amd64, linux/arm64
+          platforms: linux/amd64
```

Comment:
change back to `platforms: linux/amd64, linux/arm64`?

Reply:
sorry, read the PR description, maybe add a comment here about why `arm64` isn't being built?

Reply:
and remove the commented out line above

Reply:
👍